test/JDBC/expected/BABEL-5129-vu-verify.out

```diff
@@ -0,0 +1,131 @@
+
+-- Tests for ISNUMERIC function with varchar and nvarchar variables
+SELECT * FROM babel_5129
+GO
+~~START~~
+int#!#numeric#!#money#!#varchar#!#nvarchar
+45000#!#12345.12000#!#237891.2200#!#12.3420000000#!#12.3420000000
+~~END~~
+
+-- Test int
+SELECT ISNUMERIC(int_type)
+FROM babel_5129
+GO
+~~START~~
+int
+1
+~~END~~
+
+-- Test numeric
+SELECT ISNUMERIC(numeric_type)
+FROM babel_5129
+GO
+~~START~~
+int
+1
+~~END~~
+
+-- Test money
+SELECT ISNUMERIC(money_type)
+FROM babel_5129
+GO
+~~START~~
+int
+1
+~~END~~
+
+-- Test varchar
+SELECT ISNUMERIC(varchar_type)
+FROM babel_5129
+GO
+~~START~~
+int
+1
+~~END~~
+
+-- Test nvarchar
+SELECT ISNUMERIC(nvarchar_type)
+FROM babel_5129
+GO
+~~START~~
+int
+1
+~~END~~
+
+
+-- Test numeric variable
+DECLARE @a numeric(24,6);
+SELECT @a = 12.3420000000;
+SELECT ISNUMERIC(@a), LEN(@a), DATALENGTH(@a)
+GO
+~~START~~
+int#!#int#!#int
+1#!#9#!#6
+~~END~~
+
+
+-- Test varchar variable
+DECLARE @v varchar(20);
+SELECT @v = '12.3420000000';
+SELECT ISNUMERIC(@v), LEN(@v), DATALENGTH(@v)
+GO
+~~START~~
+int#!#int#!#int
+1#!#13#!#13
+~~END~~
+
+
+-- Test nvarchar variable
+DECLARE @nv nvarchar(10);
+SELECT @nv = '12.3420000000';
+SELECT ISNUMERIC(@nv), LEN(@nv), DATALENGTH(@nv)
+GO
+~~START~~
+int#!#int#!#int
+1#!#10#!#10
+~~END~~
+
+
+-- Test empty varchar variable
+DECLARE @v varchar(20);
+SELECT @v = '';
+SELECT ISNUMERIC(@v), LEN(@v), DATALENGTH(@v)
+GO
+~~START~~
+int#!#int#!#int
+0#!#0#!#0
+~~END~~
+
+
+-- Test empty nvarchar variable
+DECLARE @nv nvarchar(10);
+SELECT @nv = '';
```

Comment:
nice, thanks for adding this. can we try values that go beyond the limit, also NULL?

Reply:
Also, please verify the output with sql server instance.

Reply:
> Also, please verify the output with sql server instance.

Tested queries with SQL Server 16 - ISNUMERIC results match PostgreSQL. Minor unrelated discrepancy in DATALENGTH noted (PG: 6, SQL Server: 5 for 12.3420000000). Can remove DATALENGTH from the select statements in tests if needed.